README.adoc

```diff
@@ -8,9 +8,19 @@ endif::[]
 
 = Jenkins Enhancement Proposals
 
-See link:jep/1/README.adoc[JEP-1] for a description of JEPs.
-
-link:jep/[Index of all JEPs] accepted at least as drafts.
+JEP stands for Jenkins Enhancement Proposal.
+A JEP is a design document that
+describes a new feature or aspect of Jenkins itself, or Jenkins project, plugin, processes or environment.
```

Comment:
This sentence is a little hard to read, but I'm not totally sure what you were going for so I don't want to make my own edit. Maybe:
"A JEP is a design document that describes a new feature or aspect [what is an aspect?] of the core Jenkins code, a plugin, or the Jenkins project, its processes, or its environment."

Reply:
It is a direct copy from JEP-1.  The point of this change is to have something in this section rather than a bare couple of links. 

Please edit this PR to fix it in whatever way you thiink is appropriate.  I would really appreciate it.